anki/tags.py

```diff
@@ -139,13 +139,16 @@ def addToStr(self, addtags, tags):
         return self.join(self.canonify(currentTags))
 
     def remFromStr(self, deltags, tags):
-        "Delete tags if they don't exists."
+        "Delete tags if they exist."
+        def wildcard(pat, str):
+            return '*' in pat and re.search(
+                pat.replace('*', '.*'), str, re.IGNORECASE)
         currentTags = self.split(tags)
         for tag in self.split(deltags):
             # find tags, ignoring case
             remove = []
             for tx in currentTags:
-                if tag.lower() == tx.lower():
+                if (tag.lower() == tx.lower()) or wildcard(tag, tx):
```

Comment:
Fair point about `wildcard`. Though, given the scope, I don't think there's much room for confusion.

As for parens - Kernighan & Pike (1999): 

> Parentheses specify grouping and can be used to make the intent clear even when they are not required. ... [G]rouping the operands of higher-precedence operators helps the reader to see the structure more quickly.